lib/network/modules/components/edges/BezierEdgeStatic.js

```diff
@@ -4,15 +4,14 @@ import BezierEdgeBase from './util/BezierEdgeBase'
  * A Static Bezier Edge. Bezier curves are used to model smooth gradual
  * curves in paths between nodes.
  *
- * @class BezierEdgeStatic
+ * @param {Object} options
+ * @param {Object} body
+ * @param {Label} labelModule
  * @extends BezierEdgeBase
```

Comment:
jsdoc3 can't figure this out by itself?

Reply:
Sadly, no. Further, the ts generation tools require it.

Reply:
ok.